src/features/broker-protection/actions/build-url-transforms.js

```diff
@@ -20,7 +20,7 @@ export function transformUrl (action, userData) {
      * assign the updated pathname + search params
      */
     url.search = processSearchParams(url.searchParams, action, userData).toString()
-    url.pathname = processPathname(url.pathname, action, userData)
+    url.pathname = processPathname(decodeURIComponent(url.pathname), action, userData)
```

Comment:
no because then the URL constructor will re-encode it

Reply:
There is a possibility that pathname contains `%2f`. After this change, it will break the underlying code. Should we move this decoding closer to the actual templating logic?